src/main/java/org/aion/wallet/ui/components/partials/SyncStatus.java

```diff
@@ -0,0 +1,54 @@
+package org.aion.wallet.ui.components.partials;
+
+import com.google.common.eventbus.Subscribe;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.Label;
+import javafx.scene.control.ProgressBar;
+import org.aion.api.server.types.SyncInfo;
+import org.aion.wallet.WalletApi;
+import org.aion.wallet.ui.events.EventBusFactory;
+import org.aion.wallet.ui.events.TimerEvent;
+import org.aion.wallet.util.DataUpdater;
+import org.aion.wallet.util.SyncStatusFormatter;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+
+public class SyncStatus implements Initializable {
+
+    private WalletApi walletApi = new WalletApi();
+
+    @FXML
+    private ProgressBar syncProgressBar;
+
+    @FXML
+    private Label progressBarLabel;
+
+    @Override
+    public void initialize(URL location, ResourceBundle resources) {
+        registerEventBusConsumer();
+    }
+
+    private void registerEventBusConsumer() {
+        final EventBusFactory eventBusFactory = EventBusFactory.getInstance();
+        eventBusFactory.getBus(DataUpdater.FOOTER_BUS_EVENT_ID).register(this);
+    }
+
+    @Subscribe
+    private void handleConnectivityStatusEvent(TimerEvent event) {
+        SyncInfo syncInfo = walletApi.getSync();
+        setSyncBarProgress(syncInfo);
+        setSyncStatus(syncInfo);
+    }
+
+    private void setSyncStatus(SyncInfo syncInfo) {
```

Comment:
On the same theme of not using core dependencies outside the connector packages, we should get rid of `SyncInfo` references and wrap the info we need in a custom DTO object which we can use regardless of what implementation we have behind the `BlockchainConnector` - maybe `SyncStatus` would be a good name

Unfortunately this means that `SyncStatusFormatter` should also stop using `SyncInfo`